README.md

```diff
@@ -141,33 +141,46 @@ export ROS_SECURITY_ENABLE=true
 
 Now try starting another teleop node with security disabled and check that only the secure teleop node can drive the robot.
 
-# Security Workshop Sandbox Node Demo
+## Sandboxed Nodes Demo
 
 The [ROSCon 2019](https://ros-swg.github.io/ROSCon19_Security_Workshop/) [ROS2 Security Workshop](https://ros-swg.github.io/ROSCon19_Security_Workshop/)
 will present the opportunity to run the Turtlebot3 demo using the [launch-ros-sandbox](https://github.com/aws-robotics/launch-ros-sandbox)
-package. Specifically, the demo uses this package to launch the Turtlebot3 navigation nodes in a docker container. See the 
+package. Specifically, the demo uses this package to launch the Turtlebot3 navigation nodes in a docker container. See the
 configs/sandbox_demo/navigation_sandbox.launch.py launch file for details.
 
-## Running the Security Workshop Sandbox Node Demo
+### Running the Security Workshop Sandbox Node Demo
+
+``` bash
+rocker --x11 --nvidia rosswg/turtlebot3_demo "byobu -f configs/sandbox_demo/unsecure.conf attach"
+```
+
+Omit the `--nvidia` arg if you don't have dedicated GPU for hardware acceleration of 3D OpenGL views.
 
-Build the docker image required using the dockerfile from this repository
+Likewise, the following command is used to run the demo using the secure config:
 
 ``` bash
-git clone git@github.com:ros-swg/turtlebot3_demo.git
-cd turtlebot3_demo
-docker build . -t rosswg:turtlebot3_sandbox_demo
+rocker --x11 --nvidia  rosswg/turtlebot3_demo "byobu -f configs/sandbox_demo/secure.conf attach"
 ```
 
-Once the image is built, we can run it with the following command:
+### Demonstrating Sandbox Resource Limits
+
+This demo shows how robot code, acting improperly, can negatively affect the entire robotic system.
 
 ``` bash
-rocker --x11 --nvidia rosswg:turtlebot3_sandbox_demo "byobu -f configs/sandbox_demo/unsecure.conf attach"
+rocker --x11 --nvidia rosswg/turtlebot3_demo "byobu -f configs/sandbox_demo/bad_actor.conf attach"
 ```
 
-Omit the `--nvidia` arg if you don't have dedicated GPU for hardware acceleration of 3D OpenGL views.
+After launching with the above config, there is a command ready in the `bad_actor` byobu window, ready to run a cpu hog that will fork many busy processes (`ros2 run example_nodes cpu_hog 8`).
+This is a contrived example - but it illustrates what could happen if a node that you are using hits an untested code path and goes into an infinite loop.
+If you run the command, you can see the CPU of your system jump to 100% usage, grinding everything else to a halt.
 
-Likewise, the following command is used to run the demo using the secure config:
+UPCOMING: Show usage of `launch_ros_sandbox` resource limits to keep the demo running happily.
+
+## Developing
+To rebuild this demo locally if you are working on it, you can rebuild the Docker image with the same tag, so all above demo commands will work correctly.
```

Comment:
Per the comment above, this seems like a requirement to run the demo rather than solely developing.